eng/pipelines/templates/steps/build-package-artifacts.yml

```diff
@@ -94,28 +80,16 @@ steps:
     displayName: 'Install QEMU Dependencies'
     condition: and(succeeded(), eq(variables['Agent.OS'], 'Linux'))
 
-  - bash: |
-      source $(Build.SourcesDirectory)/venv/bin/activate
-      which python
-      sdk_build -d "$(Build.ArtifactStagingDirectory)" "$(TargetingString)" --service=${{parameters.ServiceDirectory}} --inactive
-    displayName: 'Generate Packages Linux/Mac'
-    condition: and(succeeded(), or(eq(variables['Agent.OS'], 'Linux'), eq(variables['Agent.OS'], 'Darwin')))
-    timeoutInMinutes: 80
-    env:
-      CIBW_BUILD_VERBOSITY: 3
-
   - pwsh: |
-      ./venv/Scripts/Activate.ps1
       which python
       sdk_build -d "$(Build.ArtifactStagingDirectory)" "$(TargetingString)" --service=${{parameters.ServiceDirectory}} --inactive
-    displayName: 'Generate Packages Windows'
-    condition: and(succeededOrFailed(), eq(variables['Agent.OS'], 'Windows_NT'))
```

Comment:
Are these condition removals intentional, or left over from testing?

Reply:
100% intentional! Totally unnecessary now that the PATH prepend from `use-venv` is working! Much simpler yaml ftw.